src/app/components/puco/puco.html

```diff
@@ -1,107 +1,76 @@
 <plex-layout main="9" *ngIf="!showPrintForm">
     <plex-layout-main>
-        <header>
-            <h3>Padrón de cobertura social</h3>
-            <div class="row">
-                <div class="col-8">
-                    <div class="buscador">
-                        <plex-text [(ngModel)]="searchTerm" name="searchTerm" (change)="buscar()"
-                                   placeholder="Ingrese DNI" [autoFocus]="autofocus" prefix="account-search">
-                        </plex-text>
-                    </div>
-                </div>
-                <div class="col-4">
-                    <plex-select *ngIf="periodos.length" [(ngModel)]="periodoSelect" name="periodoPadron"
-                                 [data]="periodos" placeholder="Seleccione un periodo"
-                                 (change)="setPeriodo($event.value)" [required]="true"></plex-select>
-                </div>
-            </div>
-        </header>
-
-        <!-- Error paciente no encontrado -->
-        <div *ngIf="usuarios && !usuarios.length && searchTerm && !loading && !errorSearchTerm && periodoSelect"
-             class="alert alert-danger">
-            <i class="mdi mdi-account-alert"></i> Paciente no encontrado en este padrón.
-        </div>
-        <!-- Error por ingreso de caracteres -->
-        <div *ngIf="errorSearchTerm && searchTerm && !loading" class="alert alert-danger">
-            <i class="mdi mdi-account-alert"></i> Este campo sólo admite caracteres numéricos.
-        </div>
+        <plex-title titulo="Padrón de cobertura social"></plex-title>
+        <plex-wrapper>
+            <plex-text type="number" [(ngModel)]="searchTerm" name="searchTerm" (change)="buscar()"
```

Comment:
Si solo acepta numeros debería ser un plex-int:

```suggestion
            <plex-int [(ngModel)]="searchTerm" name="searchTerm" (change)="buscar()"
```